Website/Packages/datax-pipeline/src/modules/flowDefinition/components/info/infoSettingsContent.jsx

```diff
@@ -73,6 +78,35 @@ export default class InfoSettingsContent extends React.Component {
             </div>
         );
     }
+
+    renderSaveButton() {
+        //Will be enabled when user has access 
+        const enableButton = this.props.saveFlowButtonEnabled;
```

Comment:
Do you use the same condition as FlowDefinitionPanel? For this particular one, should we also check if the token field is not empty?

Reply:
Added a check to make sure token field is not empty